org.eclipse.m2e.core/META-INF/MANIFEST.MF

```diff
@@ -53,12 +53,15 @@ MavenArtifact-ArtifactId: org.eclipse.m2e.core
 Import-Package: com.google.common.base;version="30.1.0",
  com.google.common.cache,
  javax.inject;version="1.0.0",
+ org.eclipse.jdt.core,
+ org.eclipse.jdt.launching,
  org.slf4j;version="[1.7.0,3.0.0)",
  org.slf4j.spi;version="[1.7.31,3.0.0)",
  com.google.gson;version="[2.9.1,3.0.0)",
  org.apache.commons.codec.digest;version="[1.14.0,2.0.0)"
 Automatic-Module-Name: org.eclipse.m2e.core
 Service-Component: OSGI-INF/org.eclipse.m2e.core.embedder.MavenModelManager.xml,
+ OSGI-INF/org.eclipse.m2e.core.internal.*.xml,
```

Comment:
Please don't use wild-cards here and just let PDE generate the explicit value.

Reply:
Not possible due to https://docs.osgi.org/specification/osgi.cmpn/8.0.0/service.component.html#d0e30931. The service from the test fragment needs to be loaded.

Reply:
no longer relevant and therefore changes reverted